packages/form-js-editor/src/features/properties-panel/entries/MultipleEntry.js

```diff
@@ -0,0 +1,43 @@
+import { get } from 'min-dash';
+
+import { ToggleSwitchEntry, isToggleSwitchEntryEdited } from '@bpmn-io/properties-panel';
+
+export function MultipleEntry(props) {
+  const { editField, field } = props;
+
+  const entries = [];
+
+  entries.push({
+    id: 'multiple',
+    component: Multiple,
+    editField: editField,
+    field: field,
+    isEdited: isToggleSwitchEntryEdited,
+    isDefaultVisible: (field) => field.type === 'filepicker',
+  });
+
+  return entries;
+}
+
+function Multiple(props) {
+  const { editField, field, id } = props;
+
+  const path = ['multiple'];
+
+  const getValue = () => {
+    return get(field, path, '');
+  };
+
+  const setValue = (value) => {
+    return editField(field, path, value);
+  };
+
+  return ToggleSwitchEntry({
```

Comment:
Wouldn't it make sense to feel enable this? It might help creating more generic forms.

Reply:
I'll do it